### PR TITLE
Create team-rotation.md

### PR DESCRIPTION
**Pull request by mweinelt**

### CODE REVIEW

File: team-rotation.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Team rotation
+
+The initial moderation team membership was defined through [RFC 102](https://github.com/NixOS/rfcs/blob/master/rfcs/0102-moderation-team.md#examples-and-interactions) and changes to the team are decided through a consensus among current moderation team members.
+
+## Joining the moderation team
+
+We are regularly on the lookout for new members, who whe think display good social skills, while interacting on the Discourse forum
```
Comment (alyssais) [inline team-rotation.md]:
```suggestion
We are regularly on the lookout for new members, who whe think display good social skills, while interacting on GitHub, on the Discourse forum
```

Maybe? Although GitHub produces a lot less need for moderation.

**Review by alyssais — APPROVED** [review]

File: team-rotation.md (hunk 1 above)
Comment (zimbatm) [reply team-rotation.md]: I think the point was more to describe where we usually source the data points from, not make it exclusive to those places. I still want to know about a God moderator even if they are on Reddit or other places.

Comment (cole-h) [reply team-rotation.md]:
```suggestion
We are regularly on the lookout for new members, who we think display good social skills, while interacting on the Discourse forum
```

Comment (mweinelt) [reply team-rotation.md]: > We are regularly on the lookout for new members, who we think display good social skills, while interacting on the community platforms.

Rewritten to be more generic.

**Comment by asymmetric** [comment]
Are there outstanding concerns,  or is this good to go? Seems reasonable to me.

**Comment by mweinelt** [comment]
I think mostly time constraints, but I think so as well. Thanks for pinging.

**Pull request merged**